externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala

```diff
@@ -82,5 +89,9 @@ class SparkSQLSessionManager private (name: String, spark: SparkSession)
     }
   }
 
+  private def stopSession(): Unit = {
+    SparkSQLEngine.currentEngine.foreach(_.stop())
```

Comment:
`sys.exit` is quite hack to exit. It might be more reasonable to use `SparkSQLEngine.stop`.